File: alignment.stringbased/src/test/java/com/vitreoussoftware/bioinformatics/alignment/suffixtree/basic/BasicSuffixTreeST.java
package com.vitreoussoftware.bioinformatics.alignment.suffixtree.basic;

import com.vitreoussoftware.bioinformatics.alignment.suffixtree.SuffixTreeFactory;
import com.vitreoussoftware.bioinformatics.alignment.suffixtree.SuffixTreeTest;

/**
 * Run SuffixTreeTest for BasicSuffixTree
 * @author John
 *
 */
public class BasicSuffixTreeST extends SuffixTreeTest {
	@Override
	protected SuffixTreeFactory getSuffixTreeFactory() {
		return new BasicSuffixTreeFactory();
	}

}

File: alignment.suffixtree/src/test/scala/com/vitreoussoftware/bioinformatics/alignment/suffixtree/basic/BasicsuffixTreeST.scala
package com.vitreoussoftware.bioinformatics.alignment.suffixtree.basic

import com.vitreoussoftware.bioinformatics.alignment.suffixtree.SuffixTreeTest


/**
  * Created by John on 8/24/14.
  */
class BasicSuffixTreeST extends SuffixTreeTest("BoundedSuffixTree") {
   def getAligner() = new BasicSuffixTreeFactory create
}
